### ISSUE TITLE
gRPC-generated code is not copied from `build/generated-proto`

### ISSUE THREAD

**Issue by alexander-yevsyukov** [SpineEventEngine/mc-java#51]
It can be reproduced when building `core-java/client`  from command line, and then open `Client.java` in IDEA. The following imports would not be resolved:

```java
import io.spine.client.grpc.CommandServiceGrpc;
import io.spine.client.grpc.CommandServiceGrpc.CommandServiceBlockingStub;
import io.spine.client.grpc.QueryServiceGrpc;
import io.spine.client.grpc.QueryServiceGrpc.QueryServiceBlockingStub;
```

Manually copying the `build/generated-proto/grpc` directory under `generated` does not make these sources visible to IDEA because source set paths were not updated to treat the `generated/grpc` directory.

**Comment by alexander-yevsyukov** [comment]
@armiol, FYI.

**Comment by armiol** [comment]
@dmdashenkov FYI. That's the same thing.

**Comment by armiol** [comment]
Addressed in #50.
